index.js

```diff
@@ -378,12 +379,28 @@ function Physics (mcData, world) {
       let inertia = physics.airborneInertia
       const blockUnder = world.getBlock(pos.offset(0, -1, 0))
       if (entity.onGround && blockUnder) {
+        let playerAttributes
+        if (entity.attributes && entity.attributes[physics.movementSpeedAttribute]) {
+          playerAttributes = entity.attributes[physics.movementSpeedAttribute]
+        } else {
+          playerAttributes = attribute.createAttributeValue(physics.playerSpeed) // default attribute
+        }
+        // Client-side sprinting (don't rely on server-side sprinting)
+        playerAttributes = attribute.deleteAttributeModifier(playerAttributes, '662a6b8d-da3e-4c1c-8813-96ea6097278d') // always delete sprinting (if it exists)
```

Comment:
what is that uuid

Reply:
SPEED_MODIFIER_SPRINTING_UUID is from LivingEntity.java